### PR TITLE
Create 0xfB7Eb1FCe8fC2Fa1F84d962191931394003C7557.setdetails.json

### PR DESCRIPTION
**Pull request by Keanuji**
ErosAM JediFund

[0x144e924d26e9f844eca75d3aa8927d8481cb75ce673870dd5c92483e226545731c2d714889053e55237cc73ce909da2c61066335d33e3098eb5c1b5266a6eaa71b]

### CODE REVIEW

File: set-details/0xfB7Eb1FCe8fC2Fa1F84d962191931394003C7557.setdetails.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "setDetails": {
+    "chainId": 1,
+    "setName": "JediFund",
+    "setSymbol": "TX3",
+    "setDecimals": 18,
+    "setDescription": "JediFund is our high-growth fund, seeking to invest in future leader crypto projects that we believe can grow at exceptional rates and provide outsized returns while keeping risk at a minimum over the long term.",
+    "website": "https://www.erosam.com",
+    "managerName": "Eros Asset Management",
+    "managerIcon": "https://i.imgur.com/RzlGGQg.png",
```
Comment (asoong) [inline set-details/0xfB7Eb1FCe8fC2Fa1F84d962191931394003C7557.setdetails.json:10]: Can you download a copy and add it to the directory so we can maintain the loading? Don't want imgur to become a dependency

**Comment by Keanuji** [comment]
Done

Thanks !

Le lun. 5 juil. 2021 à 19:14, Alex Soong ***@***.***> a
écrit :

File: set-details/0xfB7Eb1FCe8fC2Fa1F84d962191931394003C7557.setdetails.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "setDetails": {
+    "chainId": 1,
+    "setName": "JediFund",
+    "setSymbol": "TX3",
+    "setDecimals": 18,
+    "setDescription": "JediFund is our high-growth fund, seeking to invest in future leader crypto projects that we believe can grow at exceptional rates and provide outsized returns while keeping risk at a minimum over the long term.",
+    "website": "https://www.erosam.com",
+    "managerName": "Eros Asset Management",
+    "setIcon": "https://github.com/Keanuji/QindoNM/blob/main/1200px-Eros._(Greek_mythology_systematized).png",
```
Comment (asoong) [inline set-details/0xfB7Eb1FCe8fC2Fa1F84d962191931394003C7557.setdetails.json:10]: Can you check this into our repository? Don't want to be referencing another repository

**Comment by adityachoubeyy** [comment]
Aditya.choubey53@gmail.com

**Review by adityachoubeyy — APPROVED** [review]

**Review by adityachoubeyy — APPROVED** [review]
Aditya.choubey53@gmail.com

**Comment by adityachoubeyy** [comment]
Aditya.choubey53@gmail.com

**Comment by adityachoubeyy** [comment]
Trur

**Review by adityachoubeyy — CHANGES_REQUESTED** [review]
Aditya.choubey53@gmail.com

File: set-details/0xfB7Eb1FCe8fC2Fa1F84d962191931394003C7557.setdetails.json (hunk 1 above)
Comment (adityachoubeyy) [reply set-details/0xfB7Eb1FCe8fC2Fa1F84d962191931394003C7557.setdetails.json]: Sure

**Comment by justinkchen** [comment]
Closing due to inactivity

**Pull request closed without merging**